kernel/src/process_loading.rs

```diff
@@ -255,19 +149,12 @@ pub fn load_processes<C: Chip>(
     if config::CONFIG.debug_process_credentials {
```

Comment:
It didn't pretend to do credential checking before, either.

Reply:
Well, while we're here we may as well remove the thing that is incorrect

Reply:
I think the thought is that _something_ is approving credentials, because before this PR something was calling mark_credentials_accepted, and with this PR processes are getting put in the processes array, even if there is no explicit checker, so if you are debugging credentials and surprised why your processes are running, well, this is why.

Reply:
fine...